### PR TITLE
Merge 2.11 to 2.12 [ci: last-only]

### PR DESCRIPTION
**Pull request by szeiger**
Trying to figure out what's wrong with #5237

### CODE REVIEW

File: scripts/jobs/validate/test [hunk 1]
```diff
@@ -25,7 +25,9 @@ case $prDryRun in
        "partest res scalap specialized scalacheck" \
        "partest instrumented presentation" \
        "partest --srcpath scaladoc" \
+       osgiTestFelix/clean \
        osgiTestFelix/test \
+       osgiTestEclipse/clean \
```
Comment (lrytz) [inline scripts/jobs/validate/test]: @szeiger should we revert this change?

**Comment by lrytz** [comment]
thanks for performing the osgi-versions-dance!

File: scripts/jobs/validate/test (hunk 1 above)
Comment (szeiger) [inline scripts/jobs/validate/test]: Yes, I'll revert. It doesn't really hurt but should not be needed.

**Comment by szeiger** [comment]
@lrytz: It was actually you who peformed the osgi versions dance. The history of the ant build points to https://github.com/scala/scala/commit/8247b8f077c96ba9a017b53de4d8062fe428c4a7. I just copied what you did there. I still don't know _why_ the old version failed (and apparently, neither did you).

**Comment by szeiger** [comment]
Squashed (omitting https://github.com/scala/scala/pull/5242/commits/7958b3dba8b4086e93c983d66a3cb13fa46334ea)

**Comment by szeiger** [comment]
All tests green. Since this was originally @lrytz's PR, I suppose it's up to me to say: LGTM
